prog/vm/nexus.rb

```diff
@@ -176,8 +191,8 @@ def create_ipsec_tunnel(my_subnet, dst_vm, dst_subnet)
 
   def run
     host.sshable.cmd("sudo systemctl start #{q_vm}")
-    vm.update(display_state: "running")
-    hop :wait
+    hop :wait if host.sshable.cmd("sudo systemctl is-active #{q_vm}")
```

Comment:
instead of this, I want to perform an ssh access check actually.

Reply:
what do you mean?

Reply:
When we switch the vm to running state, the client still cannot ssh-in a couple of more seconds. I realized in Ozgun's demo that the couple of ssh requests failed and once the vm is up and running properly, we were able to. So, I would like to change it here so that we only switch to running after verifying that the vm is sshable.

Reply:
removed for now since it's not related to this pr